### PR TITLE
fix: use accent colour for FAB

### PR DESCRIPTION
**Pull request by simarsingh24**

### CODE REVIEW

**Comment by simarsingh24** [comment]
screenshot_20180810-162833_eventyay attendee
screenshot_20180810-162759_eventyay attendee
screenshot_20180810-162746_eventyay attendee

File: app/src/main/res/color/bottom_navigation_selector.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_checked="true" android:color="@color/colorAccent" />
+    <item android:color="@color/dark_grey"  />
```
Comment (nikit19) [inline app/src/main/res/color/bottom_navigation_selector.xml]: The default colour of the bottom navigation bar has changed

Comment (simarsingh24) [reply app/src/main/res/color/bottom_navigation_selector.xml]: updated
screenshot_1533900498

**Comment by simarsingh24** [comment]
screenshot_1533901667
After latest changes

**Pull request merged**